src/ed_optimization/application/features/order/handlers/commands/orders.py

```
@@ -0,0 +1,355 @@
+        print("Solver not available.")
+        return None
+
+    # Solve the problem
+    status = solver.Solve()
+
+    if status == pywraplp.Solver.OPTIMAL:
+        print("Solution found.")
+        groups = [[] for _ in range(group_size)]
+        for i in range(num_orders):
+            for j in range(group_size):
+                if x[i][j].solution_value() > 0.5:
```

Ruff (0.8.2)

Undefined name `x`

(F821)